source/templates.tex

```diff
@@ -3743,6 +3743,16 @@
 whether two constructs are equivalent, and they are
 functionally equivalent but not equivalent, the program is ill-formed,
 no diagnostic required.
+% FIXME: What does it mean for two function templates to correspond?
+Furthermore, if two function templates that do not correspond
+\begin{itemize}
+\item have the same name,
+\item have corresponding signatures\iref{basic.scope.scope},
+\item would declare the same entity\iref{basic.link} considering them to correspond, and
+% FIXME: What does it mean for a set of template argument lists to satisfy a function template?
```

Comment:
@opensdh: could you perhaps comment on these FIXMEs?

Reply:
1. It should presumably be "two _declarations of_ function templates", since it is declarations that correspond (and the bullet immediately above says that they declare something).
1. Is the idea that we should explicitly refer to satisfaction of the constraints?  We already have in [temp.over.link]/6.4 that a *template-head* can be "satisfied".

Reply:
Re 1): Can declarations "have the same name"? I.e. do declarations have names? Or should they "declare the same name" rather?

Reply:
Re 2): How about adding a cross-reference to [temp.over.link] right after "are satisfied"?

Reply:
Probably "introduce the same name", but maybe we should just rephrase the whole thing:
> Furthermore, if two declarations of function templates with the same name and corresponding signatures do not correspond but would declare the same entity ([basic.link]) considering them to do so, and

Reply:
Thanks! This is getting too much for the motion applicaton; I'm making this an Issue and will propose a change later!

Reply:
That's #5997.